files/fr/web/css/@font-face/ascent-override/index.html

```diff
@@ -0,0 +1,74 @@
+---
+title: ascent-override
+slug: Web/CSS/@font-face/ascent-override
+browser-compat: css.at-rules.font-face.ascent-override
+translation_of: 'Web/CSS/@font-face/ascent-override'
+---
+<p>{{CSSRef}}</p>
+
+<p>Le descripteur CSS <strong><code>ascent-override</code></strong> définit la mesure du jambage supérieur (ascendant) de la police. La mesure du jambage supérieur correspond à la distance verticale par rapport à la hauteur d'X (hauteur du caractère bas de casse, à l'exclusion des jambages inférieurs et supérieurs) que CSS doit utiliser pour agencer les boîtes dans un contexte de formatage en ligne.</p>
```

Comment:
I know it's a bit of a stretch here but I'd point to https://en.wikipedia.org/wiki/Ascender_(typography) to have a visual clue of an ascender (/me crying a bit as the wiki page for "Hauteur d'ascendante" in fr doesn't exist yet)

Reply:
Should we link the "Jambage" Wikipedia article instead?
https://fr.wikipedia.org/wiki/Jambage
(it includes both "jambage inférieur" and "jambage supérieur")

Reply:
I committed a proposal: https://github.com/mdn/translated-content/pull/1752/commits/2449a8a4f2b810a3d324962d5b115ec07840f71d
waiting for your review (no hurry!) before applying it to the other occurences (like descent-override)